test/e2e/tests/transaction/send-eth.spec.js

```diff
@@ -161,11 +161,11 @@ describe('Send ETH', function () {
           smartContract,
           title: this.test.fullTitle(),
         },
-        async ({ driver, contractRegistry }) => {
+        async ({ driver, contractRegistry, ganacheServer }) => {
           const contractAddress = await contractRegistry.getContractAddress(
             smartContract,
           );
-          await unlockWallet(driver);
+          await logInWithBalanceValidation(driver, ganacheServer);
```

Comment:
Hi 👋 , could you explain more why this change solves the flakiness? Thanks !

Reply:
hey :wave: basically this method will make sure that the balance is loaded before we proceed with any action, which is precisely the race condition that happens when this error occurs: we try to perform a tx's when the balance is still not loaded (see vid in the description).
This causes not being able to proceed with the Confirm button, and sets gas to 0, without a way of recovering from this state.

See snippet:
```
  const balance = await ganacheServer.getBalance();
  await driver.findElement({
    css: '[data-testid="eth-overview__primary-currency"]',
    text: `${balance} ETH`,
  });
```

Let me know if this clarifies :pray:

Reply:
Gotcha, thanks !